### PR TITLE
Omnipay driver rewritten to use Guzzle framework

### PR DESCRIPTION
**Pull request by steadweb**

### CODE REVIEW

**Comment by emlynwest** [comment]
Looks ok but general code style needs sorting out. Make sure your classes are imported and that the file header is above the namespace declaration.

File: src/Message/PayRequest.php [hunk 1]
```diff
@@ -0,0 +1,343 @@
...
+    {
+        return $this->setParameter('notes', $value);
+    }
+
+    /**
+     * Get the defendant first name value
+     *
+     * @return string
+     */
+    public function getDefendantFirstName()
```
Comment (emlynwest) [inline src/Message/PayRequest.php:251]: "defendant"?

Comment (steadweb) [inline src/Message/PayRequest.php:251]: Yup, the GovPayNet API requires this.

Comment (emlynwest) [inline src/Message/PayRequest.php:251]: :+1: then :)

**Comment by steadweb** [comment]
Header / Namespace declaration updated. Also made sure all classes are imported correctly.

**Comment by steadweb** [comment]
Tests are passing: https://travis-ci.org/steadweb/omnipay-govpaynet/builds/69357859

**Pull request merged**